elements/rh-timestamp/demo/rh-timestamp.html

```diff
@@ -1,4 +1,90 @@
 <link rel="stylesheet" href="demo.css">
 <script type="module" src="rh-timestamp.js"></script>
 
-<rh-timestamp></rh-timestamp>
```

Comment:
we'd prefer to split these out into separate demos. you can keep a `kitchen-sink.html` is that's easier for dev. see alert demos

Reply:
I see what you mean. Will do! :)